apps/system/js/attention_window.js

```diff
@@ -220,10 +221,25 @@
 
     // Hide on creating.
     this.notification.style.display = 'none';
+
+    this.translateNotification();
+  };
+
+  AttentionWindow.prototype.translateNotification = function() {
+    if (this.notification) {
```

Comment:
nit: just looking at ```AttentionWindow.prototype.ready/makeNotification```, it seems the general style for this file is an early return. I would opt to copy the pattern of the rest of the file to do an early return instead for consistency.